timely/src/progress/frontier.rs

```diff
@@ -157,6 +164,19 @@ impl<T: PartialOrder> Antichain<T> {
     #[inline] pub fn elements(&self) -> &[T] { &self.elements[..] }
 }
 
+impl<T: PartialEq> PartialEq for Antichain<T> {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
not sure if it works here as I haven't tried, but.. can we just do `self.elements() == other.elements()` ? slice comparison should do the job.

Reply:
It has the defect that it doesn't equate two slices that have the same elements, but in a different order. I think this was one of the reasons that I shied away from the implementation in the first place, but it's probably worth fixing (as otherwise equality testing could return surprising results anyhow).

Reply:
I assumed they were sorted, but it's not the case.. also, how can you sort incomparable elements? ahah nvm, dumb comment